code/modules/unit_tests/open_air.dm

```diff
@@ -0,0 +1,28 @@
+/datum/unit_test/open_air/Run()
+	var/datum/virtual_level/vlevel = mapzone.virtual_levels[1]
+	for(var/turf/open/T in vlevel.get_block())
+		T.Initalize_Atmos(0)
+		T.air.clear()
+	var/turf/open/center_turf = vlevel.get_center()
+	center_turf.air.set_moles(GAS_PLASMA, 32)
+	for(var/i in 1 to 10)
+		SSair.fire()
```

Comment:
Might need a sleep or tick check after each call, so that it doesn't get skipped at all, and possibly calling SSair.process_turfs() directly?